### PR TITLE
Swipe background effect

### LINKED ISSUE #609: Add animation of the CTA icon
Human Design presented an animation of the CTA where the arrow moved when hover. Could be a good addition to recover it from oblivion and implement it.

### PR DESCRIPTION
**Pull request by korgan00**
Fix #609 

Background in buttons now makes a swipe transition on hover.

### CODE REVIEW

File: components/landing/LandingCta.vue
Comment (techtolentino) [inline components/landing/LandingCta.vue:35]: this approach is very clean @korgan00 !

**Review by techtolentino — COMMENTED** [review]
@korgan00 - this looks great. I pulled down the code and tested.

Question for you or @JRussellHuffman
- looks like we decided on Option 3 from Russell's figma yeah? (if so, I like it)
- should `active` states have this same animation effect?

Great work, I can approve once we hear from Russell.

🙏

File: components/landing/LandingCta.vue
Comment (korgan00) [reply components/landing/LandingCta.vue:35]: Thank you :)

**Comment by delapuente** [comment]
Thank you @techtolentino for your review. Do not merge yet though, let's wait for @JRussellHuffman comments and another review.

**Review by delapuente — APPROVED** [review]
Thank you @korgan00, this looks nice to me.

**Comment by JRussellHuffman** [comment]
It looks great.

For future reference, would it be easy to make it animate from right to left (instead of left to right)? I like it as is because the animation goes the same direction as the arrow, but just curious how flexible the component is.

**Comment by delapuente** [comment]
Should be easy indeed, regarding the questions open by Randy, what's your take [on those](https://github.com/Qiskit/qiskit.org/pull/908#pullrequestreview-486757161)?

**Comment by JRussellHuffman** [comment]
@techtolentino to answer your questions.

- yep, looks like Option 3 (which looks good to me)
- I don't think the active state should have this same animation effect. I would imagine the animation would only get about halfway done before the state is no longer active (assuming active = mouse down).

**Comment by delapuente** [comment]
The active situation is a very good question, not for desktop, where it is mouse down indeed but thinking of mobile where you don't have mouse over, using active serves as feedback for pressing the button so the animation will play more or less depending on how long you press the button and that's provide a very, very good feeling. Let's give it a try!

File: components/landing/LandingCta.vue
Comment (delapuente) [inline components/landing/LandingCta.vue:49]: @korgan00 would you mind to give it a try and implement the same effect for the `active` state?

**Review by delapuente — COMMENTED** [review]
Let's take a look at implementing this for the `active` state so we get some feedback in mobile, following [general discussion](https://github.com/Qiskit/qiskit.org/pull/908#issuecomment-692932242).

**Pull request merged**
